docs/dev/libvirt/README.md

```diff
@@ -299,8 +299,8 @@ $ openshift-install --dir $INSTALL_DIR create manifests
 
 - Domain entry for cluster-ingress-02-config.yml file should be following (here domain is what your created initially)
```

Comment:
This line no longer makes sense with your change. Can you update it to reflect the changes you have made?

Reply:
sorry for the late response, I was on vacation.

Reply:
> I'm confused, isn't the grep command just verifying that the manifests were populated correctly? Is this sed command doing the proper replacement, even if the file is OK as is?

I just forgot to update the line before the `sed` command as @patrickdillon pointed out. I'll fix that..